pom.xml

```diff
@@ -60,38 +60,54 @@
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-		<alfresco.version>5.2.g</alfresco.version>
+		<maven.compiler.source>17</maven.compiler.source>
+		<maven.compiler.target>17</maven.compiler.target>
 
-		<!-- The following are default values for data location and Alfresco Community 
+		<!-- The following are default values for data location and Alfresco
+		Community 
 			version. Uncomment if you need to change (Note. current default version for 
-			Enterprise edition is 5.1) <alfresco.version>5.1.e</alfresco.version> <alfresco.data.location>/absolute/path/to/alf_data_dev</alfresco.data.location> -->
+			Enterprise edition is 5.1) <alfresco.version>5.1.e</alfresco.version>
+		<alfresco.data.location>/absolute/path/to/alf_data_dev</alfresco.data.location> -->
 
-		<!-- This control the root logging level for all apps uncomment and change, 
+		<!-- This control the root logging level for all apps uncomment and
+		change, 
 			defaults to WARN <app.log.root.level>WARN</app.log.root.level> -->
 
 		<!-- Set the enviroment to use, this controls which properties will be 
-			picked in src/test/properties for embedded run, defaults to the 'local' environment. 
+			picked in src/test/properties for embedded run, defaults to the 'local'
+		environment. 
 			See SDK Parent POM for more info. <env>local</env> -->
 	</properties>
 
 	<!-- Here we realize the connection with the Alfresco selected platform 
-		(e.g.version and edition) -->
+        (e.g.version and edition) -->
 	<dependencyManagement>
 		<dependencies>
-			<!-- Setup what versions of the different Alfresco artifacts that will 
-				be used (depends on alfresco.version), so we don't have to specify version 
-				in any of the dependency definitions in our POM. For more info see: http://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Importing_Dependencies -->
+			<!-- Setup what versions of the different Alfresco artifacts that
+			will be used (depends on alfresco.version),
+                 so we don't have to specify version in any of the dependency definitions in our
+			POM.
+                 For more info see:
+			http://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Importing_Dependencies
+            -->
 			<dependency>
 				<groupId>${alfresco.groupId}</groupId>
 				<artifactId>alfresco-platform-distribution</artifactId>
 				<version>${alfresco.version}</version>
 				<type>pom</type>
 				<scope>import</scope>
+				<exclusions>
+					<exclusion>
+						<groupId>org.springframework.social</groupId>
+						<artifactId>spring-social-facebook</artifactId>
+					</exclusion>
+				</exclusions>
```

Comment:
why excluding that ?

Reply:
After upgrading alfresco.version from 5.2.g I was able to compile the maven project with only  this exclusion.